src/es/components/atoms/Link.js

```diff
@@ -60,10 +60,10 @@ export default class Link extends Shadow() {
    */
   renderCSS () {
     this.css = /* css */`
-      :host > a{
+      :host > a {
         box-sizing: border-box;
         color: var(--color, red);
-        display: block;
+        display: var(--a-display, block);
```

Comment:
Body.js already uses --a-display + no one would bother to write a namespace for every <a-link> right? then Link.js would use --display and every other component would be forced to use a namespace if they don't want the same --display

but maybe i'm misunderstanding 🙌🏼

Reply:
every component which uses Link.js resp. a-link as a child forwards its namespace to it: https://github.com/mits-gossau/web-components-cms-template/blob/master/src/es/components/organisms/Footer.js#L260
if link is used directly, it is advisable to namespace it. also thats the same for other components which have the --display variable. this is in the nature of the setup. I know, that this leads to a lot of explicit variables if many components are namespaced, still I think we should stay true to the namespacing convention and try to avoid unnecessary namespacing within a component except it has multiple levels of nodes in itself. Let me know, if you got a better solution?